Helper/Requests.php

```diff
@@ -329,17 +337,15 @@ public function buildBrowserData($request = [])
     }
 
     /**
+     *
```

Comment:
maybe you forgot to write some docs for this method? or why the extra lines?

Reply:
Good catch 💪  Updated in [c4ba1d3](https://github.com/Adyen/adyen-magento2/pull/1389/commits/c4ba1d3f827be53bb3aeddde5bd63f90f5ae9cbb)